missilewars-plugin/src/main/java/de/butzlabben/missilewars/util/SetupUtil.java

```diff
@@ -82,6 +82,8 @@ public static void checkMap(String worldName) {
 
     public static void checkMissiles() {
         File file = new File(MissileWars.getInstance().getDataFolder(), "missiles");
+
+        // check if the directory "/missiles" is exist
```

Comment:
Check you grammar, it has to be either "check if ... does exist" or "check if ... exists"